src/iso.js

```diff
@@ -344,6 +365,11 @@ const renderStats = () => {
           <span class="d-block text-small text-bold">Total</span>
           <span class="d-none d-sm-block text-small text-gray-light">${datesTotal}</span>
         </div>
+        <div class="p-2 week-info">
```

Comment:
I appreciate the precision on making sure the box doesn't overlap the graph, but let's use Primer's classes here. A little more breathing room next to the box is ok.

```suggestion
        <div class="p-2 d-none d-xl-block">
```

Reply:
Looks good! Thanks 😄